packages/task/src/node/task-runner.ts

```diff
@@ -89,9 +89,14 @@ export class TaskRunnerRegistry {
      *
      * @returns the registered {@link TaskRunner} or a default runner if none is registered for the specified type.
      */
```

Comment:
Should we update the documentation to reflect the changes?

Reply:
I don't think we have to: the API should behave exactly the same as before for existing call sites.

Reply:
Ah...you mean the typedoc...

Reply:
@tsmaeder yes the typedoc, mainly to document `type` better with the changes.

> @param type the task type.

We should probably reflect that multiple `types` can be passed, and if a `runner` is present for a given type it will be chosen.

Reply:
How's this now?